src/parser/mod.rs

```diff
@@ -12012,10 +12033,26 @@ impl<'a> Parser<'a> {
                 replace_into,
                 priority,
                 insert_alias,
+                settings,
+                format_clause,
             }))
         }
     }
 
+    // Parses format clause used for [ClickHouse]. Formats are different when using `SELECT` and
+    // `INSERT` and also when using the CLI for pipes. For `INSERT` it can take an optional values
+    // list which we try to parse here.
```

Comment:
Probably no need to mention the SELECT since that's now unrelated to this one? Regarding insert and optional values, that detail I imagine is clear from the doc/type